rpm_spec_language_server/main.py

```diff
@@ -1,10 +1,20 @@
+import logging
 from rpm_spec_language_server.server import create_rpm_lang_server
 
+logging.basicConfig(format='%(levelname)s:%(funcName)s:%(message)s',
+                    level=logging.INFO)
+log = logging.getLogger()
 
 def main() -> None:
     import argparse
 
     parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--verbose", action="count", help="Verbose logging"
```

Comment:
let's not force users to type `verbose` 5 times ;-)
```suggestion
        "-v", "--verbose", action="count", help="Verbose logging"
```